### ISSUE TITLE
Preview don't work any longer if ![[image]] is inside a note

### ISSUE THREAD

**Issue by CodingFan72** [svsool/vscode-memo#401]
Hi,

after the last update of vscode it isn't possible to open a note in the preview if there is a "wiki-link" to an image inside the body like `![[image.png]]` .
The standard image link  `![](image.png)` still works.

I first thought, that it was another extension which makes the problem but I believe that the update of vscode is the reason:

I also have Vscodium on my PC. The new update just arrived today. Before the update everything was ok but after installing I had the same problem with the preview.

Here's the message I get from both apps:

```
Vorschau von Embed files.md kann nicht geöffnet werden: [UriError]: If a URI does not contain an authority component, then the path cannot begin with two slash characters ("//")
```

Regards

**Comment by svsool** [comment]
Seems to be recent security adjustments 😞. Could check when got time.

**Comment by svsool** [comment]
It was fixed in v0.3.9. Please update Memo extension.
